features/shopify-markets-js-api.md

```diff
@@ -0,0 +1,340 @@
+
+### The JS API
+In order to overcome the drawbacks of the current approach, we introduce a new JS API `migrateToShopifyMarket`. Using this API, you can load recommendations with following details updated for the current market, automatically.
+1. Product URL - updates Product URL for every product in the recommendation to match the URL of the store's market. At the same time, the URL will retain all the attribution parameters from the original URL
+2. Product Title - updates Product Title, for every product in the recommendation, to reflect the language of the current market
+3. Product Description - updates Product description, for every product in the recommendation, to reflect the language of the current market
+4. Product Category - updates Product category, for every product in the recommendation, to reflect the language of the current market
+5. Product Variant Options - For those recommendations involving chosing variants, this API translates all the variant options and title to reflect the language of the current market
+
+#### Usage
+The API takes an Object `RecProductElementSelectors` as a single parameter. The following table lists all the allowed mandatory and optional parameters of `RecProductElementSelectors` Object
+
+| Parameter Name | Required/Optional | Type | Description |
+| :-------------------: | :---------------------: | :----------: | :-------------- |
+| productSectionElement | Required | String | CSS classname/Id selector of the `div` or an equivalent wrapper element that encapsulates all the product details and the variant options  |
+| productUrlElement |  Required  | String |CSS classname selector of all <a> elements/Id of the <a> element hosting the product URL |
+| productTitleElement | Required | String | CSS classname selecror/Id of the element hosting the product title |
+| productHandleAttribute | Required | String | Attribute name for fetching the product handle. This value can be fetched in recommendation template using `$!product.lastPathOfProductUrl`. This attribute has to be part of the `div` or an equivalent wrapper element hosting the `productSectionElement` |
+| priceElement | Required | String | CSS classname selector of all elements or Id of the element displaying product `price` value. This parameter is required and is used to display the actual product price |
+| listPriceElement | Optional | String | CSS classname selector of all elements or Id of the element displaying product `compare_at_price` value. This is used only when the product has a discounted price and the recommendation displays both old & new price  |
+| listPriceElement | Optional | String | CSS classname selector of all elements or Id of the element displaying product `compare_at_price` value. This is used only when the product has a discounted price and the recommendation displays both old & new price  |
```

Comment:
This look duplicated with the line above.

Reply:
Removed!